aodncore/pipeline/files.py

```diff
@@ -863,6 +863,20 @@ def validate_unique_attribute_value(self, attribute, value):
                                                                                             value=value,
                                                                                             duplicates=duplicates))
 
+    def validate_attribute_value_matches_regexes(self, attribute, include_regexes):
+        """Check that the given :py:class:`PipelineFile` attribute matches at least one of the given regexes
+
+        :param attribute: the attribute to compare
+        :param include_regexes: list of regexes of which the attribute must match at least one
+        :return: None
+        """
+        unmatched = [f for f in self.__s if
+                     not matches_regexes(getattr(f, attribute), include_regexes=include_regexes)]
```

Comment:
I think the error message would be a bit more legible if this list only included the file name and the attribute in question. Maybe make it a dict?
```python
    unmatched = {f.name: getattr(f, attribute)
￼                 for f in self.__s
￼                 if not matches_regexes(getattr(f, attribute), include_regexes=include_regexes)}

```